shell/browser/ui/win/electron_desktop_window_tree_host_win.cc

```diff
@@ -41,23 +38,24 @@ bool ElectronDesktopWindowTreeHostWin::HasNativeFrame() const {
   // Since we never use chromium's titlebar implementation, we can just say
   // that we use a native titlebar. This will disable the repaint locking when
   // DWM composition is disabled.
-  return !ui::win::IsAeroGlassEnabled();
+  return false;
```

Comment:
Should the three-line comment above be removed? It looks like this change invalidates the comment since `return false;` here is necessary for enabling repaint locking, not disabling it:

```cc
          should_lock_(owner_->IsVisible() && !owner->HasChildRenderingWindow() &&
                       ::IsWindow(hwnd_) &&
                       !owner_->HasNativeFrame() &&
```

Reply:
This seems like it will bring back https://github.com/electron/electron/issues/1821 for windows 7

Reply:
Nice catch, I did not actually intend to change this line.